src/main/java/build/buildfarm/common/s3/S3Bucket.java

```diff
@@ -214,6 +214,7 @@ public String getAWSSecretKey() {
    * @param secretName The name where the secrets are stored.
    * @return AWS secrets for interacting with the bucket.
    */
+  @SuppressWarnings("PMD.StringInstantiation")
```

Comment:
This string use is bonkers, but we'll need the suppression anyway, because I don't know whether it is wrong or right to remove it.

Reply:
All the Base64 `byte[]` -> String examples I could find online were the same 🤯